src/ontology/mondo-edit.obo

```diff
@@ -430354,6 +430470,33 @@ is_a: MONDO:0002558 {source="Orphanet:590539"} ! melanotic neurilemmoma
 is_a: MONDO:0016752 {source="Orphanet:590539"} ! benign peripheral nerve sheath tumor
 is_a: MONDO:0018078 {source="Orphanet:590539"} ! soft tissue sarcoma
 
+[Term]
+id: MONDO:0035160
+name: progressive myoclonic epilepsy with neuroserpin inclusion bodies
+subset: ordo_subtype_of_a_disorder
+synonym: "early onset familial encephalopathy with neuroserpin inclusion bodies" EXACT [Orphanet:530298]
+xref: ICD10:G40.3 {source="Orphanet:530298", source="NTBT (ORPHA code's Narrower Term maps to a Broader Term). Attributed (The ICD10 code is attributed by Orphanet)."}
+xref: Orphanet:530298 {source="MONDO:equivalentTo"}
+is_a: MONDO:0020074 {source="https://orcid.org/0000-0001-5208-3432"} ! progressive myoclonus epilepsy
+
+[Term]
+id: MONDO:0035161
+name: progressive dementia with neuroserpin inclusion bodies
+subset: ordo_subtype_of_a_disorder
+synonym: "late-onset familial encephalopathy with neuroserpin inclusion bodies" EXACT [Orphanet:530303]
+xref: ICD10:G40.3 {source="- NTBT (ORPHA code's Narrower Term maps to a Broader Term). Attributed (The ICD10 code is attributed by Orphanet).", source="Orphanet:530303"}
```

Comment:
@matentzn is this formatted correctly?

Reply:
No, we dont want that - we need to hack this to be a source="MONDO:subClassOf" (or whatever best reflects the Orphanet source). This is also a good opportunity to change this tag (MONDO:subClassOf) to be something you can understand, like `MONDO:mondoSubClassOfSource`?

Reply:
Ok, i updated this to MONDO:mondoSubClassOfSource

Reply:
This sounds great, if you could raise this issue once at the QC call I have some more to say about it :)